Toolset/libraries/revfrontscriptlibrary.livecodescript

```diff
@@ -183,6 +183,11 @@ on nameChanged pOldname, pNewName, pTarget
    if not gREVSuppressMessages or (gREVSuppressMessages and revOKTarget()) then pass nameChanged
 end nameChanged
 
+--PM-2015-10-05: [[ Bug 16099 ]] choose pointer/browse tool does not update Tools palette
+on newtool pTool
+   send "ideToolChanged" to stack "revTools" in 0 seconds
```

Comment:
This should `pass newtool`, as we don't want the frontScript to trap the message